.travis.yml

```diff
@@ -1,10 +1,18 @@
-language: groovy
+language: nix
 sudo: false
-jdk:
-  - oraclejdk8
 cache:
   directories:
-  - $HOME/.gradle
+  - "$HOME/nix.store"
+  - "$HOME/.gradle/caches/"
+  - "$HOME/.gradle/wrapper/"
+before_install:
+- sudo mkdir -p /etc/nix
+- echo "substituters = https://cache.nixos.org/ file://$HOME/nix.store" | sudo tee -a /etc/nix/nix.conf > /dev/null
```

Comment:
Same as in https://github.com/test-editor/xtext-dropwizard/pull/45 -- is the `tee` necessary?